src/hevin/T1_Program2.java

```diff
@@ -0,0 +1,31 @@
+package hevin;
+
+public class T1_Program2 {
+
+	int charCount(String name,char ch) {
+		int count=0;
+		for(int i=0; i<name.length(); i++) {
+			if(name.charAt(i)==ch) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	void display(String name) {
+		for(int i=0;i<name.length();i++) {
+			int count = charCount(name,name.charAt(i));
+			if(count==1) {
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop